server/migrations/20200903090107_update_pushes_default_branch_name.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex) {
+	return knex.schema.alterTable('pushes', t => {
+		t
+			.string('branch')
+			.defaultTo('trunk')
+			.alter();
+	})
+};
+
+exports.down = function(knex) {
+	return knex.schema.alterTable('pushes', t => {
+		t
+			.string('branch')
+			.defaultTo('master')
+			.alter();
+	});
+};
```

Comment:
Can we merge both migrations (schema and data) together? You can't really do one without the other, as that would leave the database in inconsistent state -- default value and stored values don't match.